tests/test_binary_trie.py

```diff
@@ -258,6 +257,6 @@ def test_invalid_file():
         pytest.fail("Exception is not raised")
 
 
-def test_mutable_mapping():
+def test_mapping():
```

Comment:
I understand that, with the exception of delayed initialization (`*Trie().load`, *Trie().frombytes`), these mappings are immutable.